.github/CONTRIBUTING.md

```diff
@@ -4,16 +4,6 @@
 
 Cockatrice is written in C++-03, so do not use C++11 constructs such as `auto`.
```

Comment:
I think there's some `auto` stuff floating around already. We should definitely remove this line.